.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "editor.formatOnSave": true,
+    "editor.tabSize": 2,
+    "typescript.preferences.quoteStyle": "single"
+}
```

Comment:
Unfortunate that this file and `.prettierrc.json` overlap.

I've started putting [EditorConfig](https://editorconfig.org/) files in some of my repos.

Reply:
Until editor config supports quote type we're going to have to duplicate